setup.py

```diff
@@ -9,7 +9,7 @@
     "pgspecial>=1.11.8",
     "click >= 4.1",
     "Pygments >= 2.0",  # Pygments has to be Capitalcased. WTF?
-    "prompt_toolkit>=3.0.0,<4.0.0",
+    "prompt_toolkit>=2.0.6,<3.0.0",
```

Comment:
"prompt_toolkit>=2.0.6",

Reply:
Hi @meeuw 

Can we keep `<4.0.0` ? Otherwise it will conflict with other cli tools which requires prompt-toolkit 3.0+

https://github.com/dbcli/pgcli/issues/1148